packages/evo-blockmodels/tests/test_create.py

```diff
@@ -155,6 +182,102 @@ def setUp(self) -> None:
     def base_path(self) -> str:
         return f"blockmodel/orgs/{self.environment.org_id}/workspaces/{self.environment.workspace_id}"
 
+    def _assert_create_request(self, definition: BaseGridDefinition) -> None:
+        match definition:
+            case RegularGridDefinition():
+                size_option = models.SizeOptionsRegular(
```

Comment:
```suggestion
                size_options = models.SizeOptionsRegular(
```

Just a suggestion to update the variable name here and other places below to match the API field name, but no biggie